talosintelligence.json

```
@@ -58,7 +68,7 @@
             "action": "ip reputation",
             "identifier": "ip_reputation",
             "description": "Queries IP info",
```

More of a grammatical thing - your other actions are like "Test connectivity", "Validate the asset...", so follow same with this and the description on line 144
```suggestion
            "description": "Query IP info",
```